.azure-pipelines.yml

```diff
@@ -60,9 +60,9 @@ stages:
       androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
       iosProvisioningProfileFile: $(InternalProvisioningProfile)
       iosCertificateFile: $(InternalCertificate)
-      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
+      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS.TestFlight'
 
-- stage: AppCenter_Staging
+- stage: AppCenter_TestFlight_Staging
```

Comment:
Why do we still include `AppCenter` in the stage name?

Reply:
AppCenter is still used and can be used but it will be more limited than before.
We'll be limited to 100 iPhones/iPads.